### ISSUE TITLE
degradation of screen reader accessibility

### ISSUE THREAD

**Issue by hamadatrichine** [opnsense/core#10605]
Hello, I am a blind user using opnsense with a screen reader. I've noticed that lately accessibility is getting worse with each release.
- comboboxes / selectors broke, they used to work alright, now when you focus one and try to pick an option all you hear is section section.
- Labeling in multiple areas like interface status in interface overview, services status and their buttons to restart / stop are completely gone with the latest version for us.

These are some of the things I've  spotted, and i am  sure there are more.
The interface used to be really accessible before V26, but now it seems like there is a lack of awareness when it comes to accessibility. I know it is not a priority, and for most projects it never is, but it is a shame to see a product that was already accessible walking away from it bit by bit.
The main reason I submitted this issue is to bring awareness to the topic incase it is missing.
